apps/docs/stories/components/OrderTerminals/Docs.mdx

```diff
@@ -40,9 +37,22 @@ Component to render a form to create a new order requesting terminals.
 
 ---
 
-<Authorization
-  resources={[`read:business:business_id`, `write:account:account_id`]}
-/>
+<p>
+  Authorization is performed by passing a web component token as{' '}
+  <code>auth-token</code>.
+</p>
+<ul>
+  <li>
+    <strong>Web Component Token</strong>: These tokens are generated by your
+    backend services using the [Web Component Tokens
+    API](https://docs.justifi.tech/api-spec#tag/Web-Component-Tokens/operation/CreateWebComponentToken).
+    Each token can be scoped to perform a set number of actions and is active
+    for 60 minutes. When creating a web component token for this specific
+    component you'll need to use the roles: `read:business:business_id` and
+    `write:account:account_id`. Make sure the values for `business_id` or
+    `account_id` match the props you also pass separately.
+  </li>
+</ul>
```

Comment:
One more small request: 

The OrderTerminals component and the Checkout components are the only ones so far that require 2 roles / resources to be passed when creating the web component token. 

Can we update the formatting of this authorization section to match the way you set up the authorization section in Checkout? 

When creating a web component token for this specific component you'll need to use the following roles:

something like:

```
<ul>
  <li>
    `read:business:business_id` - use the `business_id` you pass to the component's props
  </li>
  <li>
    `write:account:account_id` - use the `account_id` you pass to the component's props
  </li>
</ul>
```